src/demo/app/app.component.ts

```diff
@@ -11,7 +11,9 @@ import { Component } from '@angular/core';
     </prx-header>
     <main>
       <article>
-        <router-outlet></router-outlet>
+        <section class="main">
```

Comment:
Publish actually has this main section in components, seemed like we could stick it here in the styleguide until we need a hero or something additional in the routed component pages.